processing/src/main/java/io/druid/segment/filter/Filters.java

```diff
@@ -119,4 +128,173 @@ public Filter apply(@Nullable String input)
 
     return filter;
   }
-}
+
+  public static Filter[] partitionWithBitmapSupport(Filter current)
```

Comment:
It's actually something more like partition() method in scala rather than filter(). But I'm ok in either way.